shell/browser/api/electron_api_system_preferences_mac.mm

```diff
@@ -193,28 +193,28 @@ AVMediaType ParseMediaType(const std::string& media_type) {
       request_id, NotificationCenterKind::kNSWorkspaceNotificationCenter);
 }
 
-int SystemPreferences::DoSubscribeNotification(
-    const std::string& name,
-    const NotificationCallback& callback,
+static NSNotificationCenter* GetNotificationCenter(
```

Comment:
Utility functions should be put into anonymous namespace without using `static`, like the `ParseMediaType` and `ConvertSystemPermission` functions in this file.

Reply:
done